### PR TITLE
fix def-set-get-param-method in rtm-ros-robot-interface.l

### PR DESCRIPTION
**Pull request by shintarokkk**
Due to recent update in roseus (https://github.com/jsk-ros-pkg/jsk_roseus/commit/46affa0c0ce1a5c13de2c7d4011295c67a54f326), a new slot named "_connection-header" was added to ros::objects.
This change has caused an error when setting parameters of rtc components from roseus (e.g. :set-impedance-controller-param).
This is because slots were parsed when creating :set-...-param methods in rtm-ros-robot-interface.l by the function def-set-get-param-method. In this function, the slot of class like hrpsys_ros_bridge::OpenHRP_ImpedanceControllerService_impedanceParam was assumed to be like (plist, m_p, d_p, ...), but after the update in roseus, it was like (plist, connection-header, m_p, d_p, ...). Therefore, additional remove of "connection-header" is necessary for making :set-...-param method.

### CODE REVIEW

File: hrpsys_ros_bridge/euslisp/rtm-ros-robot-interface.l [hunk 1]
```diff
@@ -329,7 +329,7 @@
         (debug nil))
   (when (boundp param-class)
     (let* ((param-slots-list ;; get slots list for param-class
-            (remove-if #'(lambda (x) (string= "plist" x))
+            (remove-if #'(lambda (x) (or (string= "plist" x) (string= "connection-header")))
```
Comment (k-okada) [inline hrpsys_ros_bridge/euslisp/rtm-ros-robot-interface.l]: `(string= "connection-header")` -> `(string= "connection-header" x)` ???
Sorry for trouble

Comment (shintarokkk) [reply hrpsys_ros_bridge/euslisp/rtm-ros-robot-interface.l]: Yes, I was wrong. (string= "connection-header" x) is correct.

**Comment by YoheiKakiuchi** [comment]
@shintarokkk please update this PR

**Comment by shintarokkk** [comment]
@YoheiKakiuchi Thank you for reminding, I pushed the correct version

**Pull request merged**